static/css/styles.css

```diff
@@ -381,4 +381,53 @@ p {
         width: 60px;
         color: black;
     }
+}
+
+/* Shortcuts */
+.content-padding {
+    padding: 3rem 1rem;
+}
+
+.shortcuts-container {
+    width: 100%;
+    min-width: 10vw;
+    max-width: 15vw;
```

Comment:
This is the reason for the large whitespace in the sidebar. min `5vw`, max `10vw` looks better I think

Reply:
We can update that!

The shortcuts will scroll to those header sections, and aren't links.

Reply:
Oh yes they do scroll, interesting effect!

Okay, if you push a tweak for the whitespace I'll merge it straight away

Reply:
If you can think of anything to make it obvious those scroll to their respective sections, let me know.

Reply:
I think it's fine as is, I just wasn't seeing it on that page because the whole page fits in a single browser window hence nothing happened. Not styling it as a link (different font color) seems like a good idea.